docs/source/Support/bskReleaseNotes.rst

```diff
@@ -77,6 +77,8 @@ Version |release|
   ``coastOptionBangDuration``. The setter and getter methods for this variable are renamed to reflect this change as
   ``setCoastOptionBangDuration()`` and  ``getCoastOptionBangDuration()``, respectively. See the module documentation
   for the current usage of this parameter and these associated methods.
+- Added a new N-axis spinning effector :ref:`spinningBodyNDOFStateEffector`. This is an expansion of :ref:`spinningBodyOneDOFStateEffector`
```

Comment:
Rebase on latest BSK develop and move this branch release bullet to the current beta cycle.

Reply:
Done.